crates/holochain/src/sweettest/sweet_conductor.rs

```diff
@@ -643,7 +648,7 @@ impl SweetConductor {
     }
 
     /// Start up this conductor if it's not already running.
-    pub async fn startup(&mut self) {
+    pub async fn startup(&mut self, ignore_dna_files_cache: Option<bool>) {
```

Comment:
Am I understanding correctly that this is for inline zomes support only and should be None otherwise?

Either way, this function is public and people do use Holochain as a library. I think it'd make sense to document the meaning of the parameter in the method documentation?

Reply:
There's actually also an edge case worth mentioning here. When updating coordinator zomes in a SweetConductor, the DnaFiles in that cache don't currently get updated. We'd probably need a dedicated `update_coordinators()` method on the SweetConductor` for that (but also then, we can't prevent people from using the `raw_handle()` and call `update_coordinatros()` on the actual conductor itself). Only the in-memory ribosome store will get updated and the sqlite database. So upon restarting the conductor, the ribosomes populated from the sqlite database will get overwritten by ones with the old coordinator zomes from the dna files cache on the sweet conductor by default.

Reply:
For consumers of sweettest outside of the holochain code base, it wouldn't even matter whether they set it to true or to false so I wouldn't want them to waste time trying to understand a parameter that doesn't change anything anyway for the purpose of their test.

Reply:
For Option, the default is None and for boolean the default value is false. I think I'd also prefer just a boolean if we can make that work.

Either way, I'd like the default value to be the production value. We don't fully support inline zomes and they're really for testing. Whatever the default configuration is should not need to support them.

Reply:
Code changes look good

> What do you mean with the distinction between testing and production here, given that sweettest is meant for testing?

I mean that inline zomes shouldn't be used when a user is running a real conductor. That's not intended to work and inline zomes are not intended to be a completely functional replacement for WASM. So I'm saying that if I'm setting up a Holochain-based application that uses Holochain as a library, I should be able to pick default values for everything and it should pretty much work for getting started at least. So passing `false` here would let me run WASMs and I'd only have to understand the field if I wanted to do something else.